dev-client/src/components/links/InternalLink.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright © 2024 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+import {Link} from 'native-base';
+
+type InternalLink = {
+  label: string;
+} & React.ComponentProps<typeof Link>;
+
+export default function InternalLink({label, ...props}: InternalLink) {
+  return (
+    <Link _text={{color: 'primary.main'}} isUnderlined={true} {...props}>
```

Comment:
I was a little uncomfortable directly exposing link props like this (since it means a future developer could start overriding e.g. styles on their own) however it seemed like a reasonable compromise until the usage of internal links is better understood (e.g., how to get them to link to internal webviews or other application screens, as described during design review.)

Reply:
question: is there more than one style in the figma for what we're calling InternalLink? not sure i understand why we need to expose props

Reply:
Fair enough. There isn't more than one, at least from what I can tell. So I'm okay with removing them and just re-adding whatever we need to meet the spec in the future.